shell/platform/windows/win32_flutter_window.cc

```diff
@@ -177,9 +176,7 @@ void Win32FlutterWindow::OnScroll(double delta_x, double delta_y) {
   }
 }
 
-void Win32FlutterWindow::OnClose() {
-  messageloop_running_ = false;
-}
+void Win32FlutterWindow::OnClose() {}
```

Comment:
I thought the plan was to eliminate `OnClose` entirely; it's no longer being called with the changes in this PR, right?